benchmarks/template

```diff
@@ -0,0 +1,25 @@
+2018-06-29 18:02:43
```

Comment:
This file should not be part of the PR. Please remove it.

Reply:
oops